src/config/testnet_postake_snarkless.mlh

```diff
@@ -6,3 +6,4 @@
 [%%define force_updates false]
 [%%define global_signer_real true]
 [%%define tracing true]
+[%%define consensus_mechanism "stakes"]
```

Comment:
I'd really prefer "proof_of_signature"/"proof_of_stake" or "posig"/"postake" for short. The pluralization of "sigs"/"stakes" was added by someone in reference to the tests, as in, "test_sigs" was short for "test the signature tests". I find the pluralization misleading.